src/@ndlib/gatsby-theme-marble/components/Pages/IndexPage/index.js

```diff
@@ -59,6 +59,7 @@ const IndexPage = ({ location }) => {
   const { nodes } = allMarbleItem
   return (
     <React.Fragment>
+      <p dangerouslySetInnerHTML={{__html: t('common:hompageDescriptive')}} />
       <BaseStyles>
```

Comment:
Move <p/> content into BaseStyle so that we're picking up the appropriate link color.